src/main/java/com/puppycrawl/tools/checkstyle/checks/annotation/SuppressWarningsCheck.java

```diff
@@ -242,18 +242,15 @@ public void visitToken(final DetailAST ast) {
                             case TokenTypes.QUESTION:
                                 walkConditional(fChild);
                                 break;
-                            // param in constant case
-                            // ex: public static final String UNCHECKED = "unchecked";
-                            // @SuppressWarnings(UNCHECKED)
-                            // or
-                            // @SuppressWarnings(SomeClass.UNCHECKED)
-                            case TokenTypes.IDENT:
-                            case TokenTypes.DOT:
-                                break;
                             default:
```

Comment:
Pitest will not handle cases where execution of cases match `default` case. It is basically required to merge cases with default. Since we use default, I see no reason to attach cases with default as it defeats the purpose of the default, which is all remaining items (unlisted).

I moved comments to default so information is not lost.